src/Microsoft.TestPlatform.CrossPlatEngine/Hosting/DotnetTestHostManager.cs

```diff
@@ -64,7 +65,7 @@ public class DotnetTestHostManager : ITestRuntimeProvider
         /// </summary>
         private Action<Process, string> ErrorReceivedCallback => ((process, data) =>
         {
-            if (data != null)
+            if (!string.IsNullOrEmpty(data))
             {
                 // if incoming data stream is huge empty entire testError stream, & limit data stream to MaxCapacity
                 if (data.Length > this.testHostProcessStdError.MaxCapacity)
```

Comment:
please avoid logging multiple times, `this.messageLogger.SendMessage` in *TestHostManager.cs.